apps/storefront/docs/patterns/grid-alignment/grid-alignment.mdx

```diff
@@ -5,3 +5,29 @@ mode: draft
 route: /patterns/grid-alignment/
 ---
 
+**The organization of space is key to every great design.  By aligning your component to a grid, you will provide visual structure throughout your product.**
```

Comment:
Use British English - change s to z: organisation

Reply:
Updated